src/features/pageObjects/utils/pageObjectTemplate.js

```diff
@@ -18,6 +18,12 @@ export function camelCase(n) {
 export const getClassName = (title) => {
   let className = camelCase(title);
 
+  const classNameArr = className.split(/(?=[A-Z])/);
+  if (Number(classNameArr[0])) {
+    classNameArr.shift();
+    className = classNameArr.join("");
+  }
+
```

Comment:
please, move this procedure after `if (isCyrillic(className)) {...` . Title can be cyrillic, for example, and it wouldn't work in this case

Reply:
fixed.